File: Top Interview 150/Remove Duplicates from Sorted Array/sol.js
var removeDuplicates = function (nums) {
  if (nums.length === 0) return 0;
  let c = 0;
  for (let i = 1; i < nums.length; i++) {
    if (nums[c] !== nums[i]) {
      c++;
      nums[c] = nums[i];
    }
  }
  return c + 1;
};

File: Top Interview 150/Remove Duplicates from Sorted Array/sol.go
package main

func removeDuplicates(nums []int) int {
	c := 0
	for i := 0; i < len(nums); i++ {
		if nums[c] != nums[i] {
			c++
			nums[c] = nums[i]
		}
	}
	return c + 1
}
